test/framework/cluster.go

```diff
@@ -820,7 +848,11 @@ func setEksctlVersionEnvVar() error {
 	if eksctlVersionEnv == "" {
 		err := os.Setenv(eksctlVersionEnvVar, eksctlVersionEnvVarDummyVal)
```

Comment:
Not your code, but this should almost certainly be using [e.T.Setenv](https://pkg.go.dev/testing#T.Setenv). If you feel like tweaking while you're there.